core/lib/dal/src/consensus_dal.rs

```diff
@@ -554,6 +554,7 @@ impl ConsensusDal<'_, '_> {
         Ok(Some(AttestationStatus {
             genesis: genesis.hash(),
             next_batch_to_attest,
+            consensus_registry_address: None, // Assigned in the API layer.
```

Comment:
main node also needs to know this address, while it cannot query api currently. How do we resolve this?

Reply:
My potentially flawed understanding is by adding it to [the config proto](https://github.com/matter-labs/zksync-era/pull/2725/files#diff-9c82e3284daf363ee808d461fc108a1801ddcdfcfb8e00f23ee491d310ea6a69R24) it will read the value which is written by https://github.com/matter-labs/zksync-era/pull/2713 

```
cat ./chains/era/configs/contracts.yaml | yq .l2.consensus_registry
0x72ada8c211f45e768c9a7781793da84daf1d0d1b
```

This is then read when you use the `zk_toolbox` to [start the server](https://github.com/matter-labs/zksync-era/blob/c162510598b45dc062c2c91085868f8aa966360e/zk_toolbox/crates/zk_inception/src/commands/server.rs#L58)

I also assume that on the kubernetes config this would be something we have to add to some config file manually after the contract has been deployed, for example [here](https://github.com/matter-labs/gitops-kubernetes/blob/f29949bc16ce5072df66c797b6dc2d114d5faa8b/apps/environments/stage/server-v2-config/contracts.yaml#L11).

Reply:
ok, so you are suggesting to just propagate it to the consensus component from the contracts config. I suppose that's ok.